src/runner.py

```diff
@@ -303,17 +311,67 @@ def post_to_onboarding_lights(self, data):
                 requests.exceptions.Timeout
             ) as connection_error:
                 log(
-                    f'Failed to post onboarding lights to {ONBOARDING_LEDS_API}: {data}\n'
+                    f'Failed to post onboarding lights to {ONBOARDING_LEDS_API}/api/'
+                    f'trigger: {data}\n'
                     f'{str(connection_error)}'
                 )
                 sentry_sdk.capture_exception(connection_error)
             except requests.HTTPError as exception:
                 log(
-                    f'Failed to trigger onboarding lights {ONBOARDING_LEDS_API} '
+                    f'Failed to trigger onboarding lights {ONBOARDING_LEDS_API}/api/trigger '
                     f'with error {str(exception)}'
                 )
                 sentry_sdk.capture_exception(exception)
 
+    def onboarding_authentication_daemon(self):
+        """
+        If ONBOARDING_LEDS_TOKEN_API is set, start a daemon to update
+        authentication token before it expires
+        """
+        if ONBOARDING_LEDS_API:
+            while True:
+                self.update_onboarding_authentication()
+                sleep(30)
+
+    def update_onboarding_authentication(self):
+        """
+        Update onbording lens token if it is about to expire
+        """
+        now = time()
+        time_till_expiry = self.onboarding_authentication_expiry_time - now
+
+        # less then 45 seconds to expiry
+        if time_till_expiry < 45:
```

Comment:
```suggestion
        # less then 240 seconds to expiry
        if time_till_expiry < 240:
```
To avoid hammering the lights api.